code/__DEFINES/obj_flags.dm

```diff
@@ -50,6 +50,8 @@
 #define SCAN_BOOZEPOWER         (1<<12) //! Allows helmets and glasses to scan reagents.
 #define MASKEXTENDRANGE			(1<<13) //! For masks, allows you to breathe from internals on adjecent tiles
 #define NOTCONSUMABLE			(1<<14) //! Moths cannot eat clothing with that flag
+/// Usable as casting clothes by wizards (matters for suits, glasses and headwear)
+#define CASTING_CLOTHES (1<<15)
```

Comment:
Credit the author

https://github.com/tgstation/tgstation/pull/66604

Reply:
Didn't even know this was a pr
Nevertheless I will, thanks for the heads up